gspread_formatting/models.py

```diff
@@ -187,11 +189,29 @@ def __init__(self, themeColor=None, rgbColor=None):
 class Color(CellFormatComponent):
     _FIELDS = ('red', 'green', 'blue', 'alpha')
 
-    def __init__(self, red=None, green=None, blue=None, alpha=None):
+    def __init__(self, red=None, green=None, blue=None, alpha=None, hexcolor=None):
```

Comment:
No need to add `hexcolor` paramter to the constructor. `fromHex` should do all the work.